src/patternfly/components/EmptyState/empty-state.scss

```
@@ -55,8 +55,8 @@
   --#{$empty-state}--m-xs__footer--MarginBlockStart: var(--pf-t--global--spacer--md);
 
   // Actions
-  --#{$empty-state}__actions--RowGap: var(--pf-t--global--spacer--sm);
-  --#{$empty-state}__actions--ColumnGap: var(--pf-t--global--spacer--md);
+  --#{$empty-state}__actions--RowGap: var(--pf-t--global--spacer--action--vertical--default);
```

this token is meant for the padding within actions, should probably use a gap token here instead... maybe --pf-t--global--spacer--gap--group--vertical?
```suggestion
  --#{$empty-state}__actions--RowGap: var(--pf-t--global--spacer--gap--group--vertical);
```